### PR TITLE
Update sql-agent-extension-manually-register-single-vm.md

### PR DESCRIPTION
**Pull request by PhilB-MSFT**
Adding note that registration for VMs from PowerBI will be blocked.

### CODE REVIEW

**Comment by ShannonLeavitt** [comment]
@adbadram 

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: azure-sql/virtual-machines/windows/sql-agent-extension-manually-register-single-vm.md [hunk 1]
```diff
@@ -46,6 +46,9 @@ To register your SQL Server VM with the extension, you'll need:
 - The latest version of [Azure CLI](/cli/azure/install-azure-cli) or [Azure PowerShell (5.0 minimum)](/powershell/azure/install-az-ps).
 - A minimum of .NET Framework 4.5.1 or later.
 
+   > [!NOTE]
+   > Creating SQL Virtual Machine Resource for PowerBI VM images is not supported.
```
Comment (rwestMSFT) [inline azure-sql/virtual-machines/windows/sql-agent-extension-manually-register-single-vm.md]:
```suggestion
   > Creating a SQL Virtual Machine Resource for Power BI VM images is not supported.
```

Comment (PhilB-MSFT) [reply azure-sql/virtual-machines/windows/sql-agent-extension-manually-register-single-vm.md]: Reads much better, thank you.

**Comment by MashaMSFT** [comment]
hey @PhilB-MSFT , thanks for your submission! I modified the text and added it to a different doc in this PR: https://github.com/MicrosoftDocs/sql-docs-pr/pull/27250  so I'll go ahead and close this one out :) Thank you!!!!!

**Pull request closed without merging**